compiler/src/iree/compiler/Codegen/VMVX/Passes.cpp

```diff
@@ -58,6 +58,10 @@ void addVMVXDefaultPassPipeline(OpPassManager &funcPassManager,
     funcPassManager.addPass(
         createCPULowerToUKernelsPass(clSkipIntermediateRoundings));
   }
+  {
+    FunctionLikeNest newFuncPassManager(funcPassManager);
+    addEncodingToNopPasses(newFuncPassManager);
+  }
```

Comment:
I think we don't need the change for VMVX side either. It should already be handled by CPUMaterializeEncoding pass.

https://github.com/iree-org/iree/blob/d174e8bcec9f221082511c67111b3f995bdd54a0/compiler/src/iree/compiler/Dialect/VMVX/Transforms/Passes.cpp#L42-L51

Note: The VMVX pipeline is quite different from other backends, so you don't find the pass in this file.